roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -126,6 +126,9 @@
     dest: "{{ kube_config_dir }}/kubeadm-config.{{ kubeadmConfig_api_version }}.yaml"
   register: kubeadm_config
 
+- name: Pulling kubernetes images
```

Comment:
Isn't this a task for the download role?

Reply:
There is no good way, it needs to rely on the --config file